arch/xtensa/Kconfig

```diff
@@ -93,6 +93,32 @@ config XTENSA_NUM_SPIN_RELAX_NOPS
 	  Specify the number of NOPs in Xtensa specific
 	  arch_spin_relax().
 
+# These are selected by each SoC according to their core-isa.h
+config XTENSA_ARCH_HAS_HIFI1
+       bool
+       help
+         HiFi1 Audio Engine
+
+config XTENSA_ARCH_HAS_HIFI2
+       bool
+       help
+         HiFi2 Audio Engine
+
+config XTENSA_ARCH_HAS_HIFI3
+       bool
+       help
+         HiFi3 Audio Engine
+
+config XTENSA_ARCH_HAS_HIFI4
+       bool
```

Comment:
can we add more to make this be true or false, like below:

default y if SOC_INTEL_ACE15_MTPM || SOC_INTEL_ACE20_LNL

or you want to add similar in other place?
I mean we need have a place to enable these config.

Reply:
No `default` is supposed to be here https://docs.zephyrproject.org/latest/build/kconfig/tips.html#using-select-for-helper-symbols

Reply:
@andyross @dbaluta @kv2019i Can we say that all SOF users are now go with zephyr even for IPC3? 
i.e, all existing SOF users are depend on zephyr?

Reply:
> @andyross @dbaluta @kv2019i Can we say that all SOF users are now go with zephyr even for IPC3?
> i.e, all existing SOF users are depend on zephyr(no matter IPC3/IPC4)?
> AMD, MTK, NXP are all depend on zephyr now?

For NXP, all platforms are supported both with Zephyr and XTOS  (IPC3-only, so far).

Reply:
Thanks, Daniel, so this year means 2024, right?
I think it is quite clear now: at least, NXP still have XTOS based IPC3 build, which means we still can't fully depend on zephyr to pass HIFIn information to SOF.